docs/ops.html

```diff
@@ -1247,7 +1247,168 @@ <h4 class="anchor-heading"><a id="prodconfig" class="anchor-link"></a><a href="#
+  </p>
+  
+  <p><b>Example <code>DirectoryConfigProvider</code> configuration</b></p>
+  <pre><code>config.providers=dirProvider
+config.providers.dirProvider.class=org.apache.kafka.common.config.provider.DirectoryConfigProvider
+config.providers.dirProvider.param.dir=/path/to/config/directory</code></pre>
+  
+  <h4 class="anchor-heading">
+    <a id="env_var_config_provider" class="anchor-link"></a>
+    <a href="#env_var_config_provider">EnvVarConfigProvider</a>
+  </h4>
+  
+  <p>The <code>EnvVarConfigProvider</code> retrieves configuration data from environment variables.</p>
+  <p>No specific parameters are required, as it reads directly from the specified environment variables.</p>
+  <p>
+    This provider is useful for configuring applications running in containers, for example, to load certificates or JAAS configuration from environment variables mapped from secrets.
+  </p>
+  
+  <p><b>Example <code>EnvVarConfigProvider</code> configuration</b></p>
+  <pre><code>config.providers=envVarProvider
+config.providers.envVarProvider.class=org.apache.kafka.common.config.provider.EnvVarConfigProvider</code></pre>
+  
+  <h4 class="anchor-heading">
+    <a id="file_config_provider" class="anchor-link"></a>
+    <a href="#file_config_provider">FileConfigProvider</a>
+  </h4>
+  
+  <p>The <code>FileConfigProvider</code> retrieves configuration data from a single properties file.</p>
+  <p>You specify the path to the properties file using the <code>file</code> parameter.</p>
+  <p>This provider is useful for loading configuration data from mounted files.</p>
+  
+  <p><b>Example <code>FileConfigProvider</code> configuration</b></p>
+  <pre><code>config.providers=fileProvider
+config.providers.fileProvider.class=org.apache.kafka.common.config.provider.FileConfigProvider
+config.providers.fileProvider.param.file=/path/to/config.properties</code></pre>
+  
+  <h4 class="anchor-heading">
+    <a id="ref_config_provider" class="anchor-link"></a>
+    <a href="#ref_config_provider">Referencing configuration providers</a>
+  </h4>
+  <p>
+    To reference a property value supplied by a configuration provider, use the correct placeholder syntax:
+  </p>
+  <dl>
+    <dt><b><code>DirectoryConfigProvider</code></b></dt>
+    <dd>
+      <p><code>${dirProvider:&lt;path_to_file&gt;:&lt;file_name&gt;}</code></p>
+    </dd>
+    <dt><code><b>EnvVarConfigProvider</code></b></dt>
+    <dd>
+      <p><code>${envVarProvider:&lt;enVar_name&gt;}</code></p>
+    </dd>
+    
+    <dt><b><code>FileConfigProvider</code></b></dt>
+    <dd>
+      <p><code>${fileProvider:&lt;path_and_filename&gt;:&lt;property&gt;}</code></p>
+    </dd>
+  </dl>
+  <p>
+    Here’s an example that uses a file configuration provider with Kafka Connect to provide authentication credentials to a database for a connector.
+  </p>
+  <p>
+    First, create a <code>connector.properties</code> configuration file with the following credentials:
```

Comment:
Should we name that file `credentials.properties`, or `connector-credentials.properties` so it's not confused with the actual connector configurations that can also be provided as a file.